vyxal/helpers.py

```diff
@@ -919,17 +919,13 @@ def vy_map(function, vector, ctx: Context = DEFAULT_CTX):
 
 def vyxalify(value: Any) -> Any:
     """Takes a value and returns it as one of the four types we use here."""
-    if isinstance(value, sympy.core.numbers.Integer):
-        return int(value)
+    if isinstance(value, bool):
+        return str(value)
```

Comment:
Hang on... shouldn't booleans be converted to 0/1?

Reply:
No, because this is making sure that if you input True or False, they're strings instead of numbers.

https://vyxal.pythonanywhere.com/#WyIiLCIiLCIxKyIsIiIsIlRydWUiXQ==

Reply:
Oh, okay.